modules/network-observability-enriched-flows-kafka.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: Procedure
+[id="network-observability-enriched-flows-kafka_{context}"]
+= Export enriched network flow data
+
+You can store network flows in Kafka. 
+
+.Prerequisites
+* Installed Kafka
+* The `FlowCollector` spec.agent is specified as `Kafka`.
+
+.Procedure
+
+. Edit the `FlowCollector` to configure `spec.exporters` as follows:
++
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+exporters:
+    - type: KAFKA
+      kafka:
+        address: "kafka-cluster-kafka-bootstrap.netobserv"
+        topic: netobserv-flows-export   <1>
+----
+<1> NetObserv exports all flows to the Kafka server on the provided topic name
+
+. Next, flows data are sent to an available output, such as Splunk, Elastic, or FluentID, in a JSON format. See the next topic, Network Flows format specifications for more information. 
```

Comment:
Maybe `flows data can be sent` rather than `flow data are sent` (else, it suggests that this is something NetObserv does on its own, which it doesn't.
Also, it's `Fluentd` not `FluentID`.
@stleerh any opinion on which examples of output we can provide there? I think it's good to mention splunk or elastic, maybe others as well, it could be driven by what we hear customers are interested in

Reply:
Sure, makes sense to me. I can see the difference. Also, 🤦‍♀️ to adding the "I". Thanks. 
Also is Elastic just Elastic or is it Elasticsearch? 

I think the RFE for this epic specifically mentions that the customer wishes to use splunk. I guess, are next steps here to provide JSON formatting that is catered to a specific output of our choosing?